http_server.py

```diff
@@ -0,0 +1,107 @@
+import socket
+import os
+import sys
+
+
+ROOT = "./webroot/"
+ADDR = ("127.0.0.1", 8000)
+server = socket.socket(
+    socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_IP
+)
+server.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+
+
+def resolve_uri(uri):
+    if "../" in uri:
+        raise UserWarning("403 Forbidden")
+
+    if os.path.isfile(uri):
+        TYPE = uri.split(".")[-1].lower()
+        if TYPE in ('PNG', 'JPG', 'JPEG', 'GIF'):
```

Comment:
how is a type that has been converted to lowercase ever going to match an item in this tuple?